WORKSPACE

```diff
@@ -33,14 +33,14 @@ bind(
 # 1. Determine SHA256 `wget https://github.com/envoyproxy/envoy/archive/$COMMIT.tar.gz && sha256sum $COMMIT.tar.gz`
 # 2. Update .bazelversion, envoy.bazelrc and .bazelrc if needed.
 #
-# Commit date: 2024-05-16
```

Comment:
I just fixed the sed pattern in the `maistra/scripts/update-envoy-sha.sh` script so that it updates the date as well as the envoy commit & sha, as intended. Therefore, the date format now comes from that script, rather than being hand edited as it was previously

Reply:
gotcha. I guess if we change the initialization of the `$today` variable to use `date +%F` it would be the same format as before